Python/pyintrinsics.c

```diff
@@ -0,0 +1,18 @@
+#include "Python.h"
+
+#ifndef HAVE_BIT_LENGTH
+static const unsigned char BitLengthTable[32] = {
+    0, 1, 2, 2, 3, 3, 3, 3, 4, 4, 4, 4, 4, 4, 4, 4,
+    5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5
+};
+
+int bit_length(unsigned long d) {
```

Comment:
Please rename the function to _Py_bit_length().

Why the result is signed? Why not unsigned int?